### PR TITLE
initial supabase db

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Initialize the database information

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3156** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3156" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3156?utm_source=stack-comment).

File: supabase/config.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+[storage]
+# The maximum file size allowed (e.g. "5MB", "500KB").
+file_size_limit = "50MiB"
+
+[auth]
+# The base URL of your website. Used as an allow-list for redirects and for constructing URLs used
+# in emails.
+site_url = "http://localhost:3000"
+# A list of *exact* URLs that auth providers are permitted to redirect to post authentication.
+additional_redirect_urls = ["https://localhost:3000"]
```
Comment (tomquirk) [inline supabase/config.toml:45]: **question** whats the go here for prod?

**Review by tomquirk — APPROVED** [review]

File: supabase/config.toml (hunk 1 above)
Comment (wraeth-eth) [reply supabase/config.toml:45]: good eye - uaffected for prod. this is just for local

**Comment by wraeth-eth** [comment]
@wraeth-eth queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3156).

**Comment by wraeth-eth** [comment]
@wraeth-eth merged this pull request with [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3156).

**Pull request merged**